core/services/keystore/master.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/smartcontractkit/sqlx"
 )
 
+//go:generate mockery --name Master --output ../../internal/mocks/ --case=underscore
```

Comment:
We're moving away from the idea of "put all the mocks in one folder" and instead we're trying to keep mocks scoped by domain. Can you move these to `core/services/keystore/mocks/` :pray:.